### DISCUSSION TITLE
Journey Mode Auto Research?

### DISCUSSION THREAD

**Discussion by Cobalt24** [Pryaxis/TShock#1984] [Q&A] [answered]
Hey all, I'm looking for a way to give SSCs access to a fully researched item duplicator menu from the moment they join. Any way to do this, plugins or otherwise?

Apologies if this is included in any documentation, I've read it over a few times and couldn't find anything related to it. If nothing exists for this yet, I think it would be a cool option to include especially for server admins! Would really help cut out the time that may be wasted typing out item spawn commands.

Any help would be appreciated! Thank you!

**Comment by hakusaro** [comment, marked as answer]
It's possible to do but a plugin is required.

**Reply by Cobalt24** [reply]
Is there anywhere I can look for plugins or learn to make my own?
